src/from-markdown.ts

```diff
@@ -65,11 +65,14 @@ const exit = {
 
 // Bindings
 function enterBindingContent (this: CompileContext, token: Token) {
+  const bindingValue = this.sliceSerialize(token).split('|');
```

Comment:
For me, I prefer to use `||` instead of single `|`. This is exactly what you can use in Vue template `{{ value || 'default' }}`

WDYT?

Reply:
@farnabaz I've been using Twig syntax out of habit, but I agree that the Vue syntax is cleaner. Also, I'm not entirely sure about the proper way to define the start and end of quotes. Could you share some insights on that? Thanks!